### PR TITLE
Palak Assignment 8 GIT hands On

### PR DESCRIPTION
**Pull request by PalakSJun20**
Assignment 8 Employee .

### CODE REVIEW

File: src/palakSJun20/Employee.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+	public static void main(String[] args) {
+		Employee employee1 = new Employee(); // Employee 1 
+		employee1.setData(44561, "Palak", 35000, 4);
+		System.out.println("--Employee 1 Details--");
+		System.out.println(" Name : " + employee1.empName);
+		System.out.println(" Id : "+ employee1.empId);
+		System.out.println(" Employee Band : "+ employee1.getEmployeeBand());
+		System.out.println(" Department : "+ employee1.getEmployeeDeptName());
+
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/Employee.java]: Remove extra space.

File: src/palakSJun20/Employee.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+		System.out.println(" Name : " + employee1.empName);
+		System.out.println(" Id : "+ employee1.empId);
+		System.out.println(" Employee Band : "+ employee1.getEmployeeBand());
+		System.out.println(" Department : "+ employee1.getEmployeeDeptName());
+		
+		if(employee1.isEligibleForIncrement())
+			System.out.println(" Employee is eligible for increment");
+		else
+			System.out.println(" Not eligible for increment");
+
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/Employee.java]: Same for here.

File: src/palakSJun20/Employee.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+		System.out.println(" Employee Band : "+ employee1.getEmployeeBand());
+		System.out.println(" Department : "+ employee1.getEmployeeDeptName());
+		
+		if(employee1.isEligibleForIncrement())
+			System.out.println(" Employee is eligible for increment");
+		else
+			System.out.println(" Not eligible for increment");
+		
+		Employee employee2 = new Employee();
+
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/Employee.java]: Same for here.

File: src/palakSJun20/Employee.java [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+		System.out.println("\n--Employee 2 Details--");
+		System.out.println(" Name : " + employee2.empName);
+		System.out.println(" Id : "+ employee2.empId);
+		System.out.println(" Employee Band is : "+ employee2.getEmployeeBand());
+		System.out.println(" Department : "+ employee2.getEmployeeDeptName());
+		if(employee2.isEligibleForIncrement())
+			System.out.println(" Employee is eligible for increment");
+		else
+			System.out.println(" Not eligible for increment");
+
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/Employee.java:78]: Same for here.

File: src/palakSJun20/Employee.java [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+		System.out.println(" Id : "+ employee2.empId);
+		System.out.println(" Employee Band is : "+ employee2.getEmployeeBand());
+		System.out.println(" Department : "+ employee2.getEmployeeDeptName());
+		if(employee2.isEligibleForIncrement())
+			System.out.println(" Employee is eligible for increment");
+		else
+			System.out.println(" Not eligible for increment");
+		
+	}
+
```
Comment (KrishnaKTechnocredits) [inline src/palakSJun20/Employee.java:80]: Same for here.

File: src/palakSJun20/Employee.java (hunk 3 above)
Comment (PalakSJun20) [reply src/palakSJun20/Employee.java]: Done

File: src/palakSJun20/Employee.java (hunk 2 above)
Comment (PalakSJun20) [reply src/palakSJun20/Employee.java]: done

File: src/palakSJun20/Employee.java (hunk 1 above)
Comment (PalakSJun20) [reply src/palakSJun20/Employee.java]: done

**Pull request merged**